odl/tomo/util/utility.py

```diff
@@ -82,7 +82,11 @@ def euler_matrix(*angles):
              sth * cps,
              cth]])
 
-    return mat
+    if np.isscalar(angles[0]):
+        return mat.squeeze()
+    else:
+        # Move third axis to the beginning
```

Comment:
We want `mat[0]` to be the first matrix etc. But you're right, the comment should say that rather than state the obvious.